src/markdown-pages/install-with-kurl/index.md

```diff
@@ -23,13 +23,27 @@ HA installs will prompt and wait for an optional load balancer address to be pro
 
 ### Converting to HA (Beta)
 
-To convert a non-HA cluster to an HA cluster, re-run the install script with the `ha` flag.
-If the cluster has remote secondary nodes, the install script will print out a command that must be run on every node to update clients to use the new load balancer address.
+To convert a non-HA cluster to an HA cluster, re-run the install script with the `ha` flag:
 
-```
+```bash
 curl https://kurl.sh/latest | sudo bash -s ha
 ```
 
+If the cluster has remote nodes, the install script will print out a command that must be run on every node to update clients to use the new load balancer address:
+
+```bash
+curl https://kurl.sh/latest/tasks.sh | sudo bash -s set-kubeconfig-server https://loadbalancer.internal:6443
```

Comment:
Will it always be this literal command? Or will the URL change based on the LB address?

Reply:
It will vary based on the LB address

Reply:
we should have it more clearly be variable in these docs then IMO

Reply:
Maybe `https://k8slb.somebigbank.com:6443` @areed ?

Reply:
Sounds good